src/autoplex/data/common/jobs.py

```diff
@@ -699,6 +699,7 @@ def preprocess_data(
     regularization: bool = False,
     retain_existing_sigma: bool = False,
     scheme: str = "linear-hull",
+    element_order: list = None,
```

Comment:
```suggestion
    element_order: list | None = None,
```
list = None would cause a data type mismatch for list